docs/data/material/guides/next-js-app-router/next-js-app-router.md

```diff
@@ -0,0 +1,227 @@
+# Next.js App Router
+
+<p class="description">Learn how to use MUI libraries with the Next.js App Router</p>
+
+## Next.js and React Server Components
+
+Next.js's App Router implements React Server Components, which is a [new feature](https://github.com/reactjs/rfcs/blob/main/text/0227-server-module-conventions.md#changes-since-v1) in React.
```

Comment:
```suggestion
The Next.js App Router implements React Server Components, a [new feature](https://github.com/reactjs/rfcs/blob/main/text/0227-server-module-conventions.md#changes-since-v1) introduced in React 18.
```